src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/ActivatorUtilities.cs

```diff
@@ -169,7 +169,7 @@ public static object CreateInstance(
             Type?[] argumentTypes;
             if (parameters.Length == 0)
             {
-                argumentTypes = Type.EmptyTypes;
+                argumentTypes = [];
```

Comment:
Once this change is picked up by ASP.NET, kestrel-minimal-linux and some other numbers should improve too. (rt-sz does not seem to be picking up dotnet/runtime package updates.)